docs/fundamentals/toc.yml

```
@@ -2790,23 +2790,25 @@ items:
       href: ../standard/parallel-programming/task-parallel-library-tpl.md
       items:
       - name: Data parallelism
-        href: ../standard/parallel-programming/data-parallelism-task-parallel-library.md
         items:
-        - name: "How to: Write a Simple Parallel.For Loop"
+        - name: Overview
+          displayName: data parallelism
+          href: ../standard/parallel-programming/data-parallelism-task-parallel-library.md
+        - name: Write a simple Parallel.For loop
           href: ../standard/parallel-programming/how-to-write-a-simple-parallel-for-loop.md
-        - name: "How to: Write a Simple Parallel.ForEach Loop"
+        - name: Write a simple Parallel.ForEach loop
```

Should we remove "simple" per the style guide? (I don't have strong feelings, but wanted to mention it).

```suggestion
        - name: Write a Parallel.ForEach loop
```